madgiraffestudios/src/concept.txt

```diff
@@ -0,0 +1,39 @@
+import Carousel from 'react-bootstrap/Carousel'
```

Comment:
- [ ] Remove file, don't think this is needed

Reply:
will resolve in the next PR